src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -33,6 +33,7 @@ object JsonRpcController {
     val Net = "net"
     val Db = "db"
     val Personal = "personal"
+    val Daedalus = "daedalus"
```

Comment:
:+1: This has already been done by Lukasz so after updating up to latest `phase/daedalus` it shouldn't be marked as a change